cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForStaticVarCompensators.java

```diff
@@ -123,10 +123,10 @@ private void setRegulatingControl(RegulatingControl control, StaticVarCompensato
         svc.setReactivePowerSetPoint(targetReactivePower);
         svc.setRegulationMode(regulationMode);
 
-        control.hasCorrectlySetEq(svc.getId());
+        control.hasCorrectlySet();
```

Comment:
Same remark as above. This method should only indicates that it has been correctly set **for this equipment**.

Reply:
Refactored according with your comments.